main.py

```diff
@@ -293,6 +299,48 @@ def disable_switch(self, request: Request) -> JSONResponse:
         self.notify_switch_links_status(switch, "link disabled")
         return JSONResponse("Operation successful", status_code=201)
 
+    @rest('v3/switches/{dpid}', methods=['DELETE'])
+    def delete_switch(self, request: Request) -> JSONResponse:
+        """Delete switch"""
+        dpid = request.path_params["dpid"]
+        endpoint = settings.FLOW_MANAGER_URL +\
+            f'/stored_flows?state=installed&dpid={dpid}'
+        response = httpx.get(endpoint)
```

Comment:
Method created `get_flows_by_switch()`

Reply:
Overall, looks good. Retries haven't been handled though if you address it. Retries aren't as critical here since this only being called via a user endpoint `DELETE v3/switches/{dpid}`, so users can themselves call again without losing data, but still it'd be great to have retries here too.

Reply:
Added retries